reports/sources/issues.sql

```diff
@@ -1 +1 @@
-select * from issue_events
+select * from octocatalog.issue_events
```

Comment:
nice! interesting it worked locally.
Thanks @gwenwindflower

Reply:
yea i think locally DuckDB make the assumption that you're using `main` schema if you don't specify, whereas MotherDuck requires a specification (i _think_ that's what's going on)